spk/octoprint/src/requirements.txt

```diff
@@ -1,35 +1,36 @@
-argh==0.26.1
-awesome-slugify==1.6.5
-Babel==2.0
-backports.ssl-match-hostname==3.4.0.2
+# taken from setup.py
+flask==0.12.5
+Jinja2==2.8.1
+tornado==4.5.3
+regex==2020.7.14
+
+# anything below this should be checked on releases for new versions
+Flask-Login==0.4.1
+Flask-Babel==0.12.2
+Flask-Assets==0.12
 blinker==1.4
-certifi==2015.9.6.2
-Flask==0.10.1
-Flask-Assets==0.10
-Flask-Babel==0.9
-Flask-Login==0.2.2
-Flask-Markdown==0.3
-Flask-Principal==0.3.5
-itsdangerous==0.24
-Jinja2==2.8
-Markdown==2.6.2
-netaddr==0.7.17
-netifaces==0.10.4
-pathtools==0.1.2
-pkginfo==1.2.1
-pyasn1==0.1.8
-pylru==1.0.9
-pyserial==2.7
-pytz==2015.6
-regex==2016.03.02
-requests==2.7.0
-rsa==3.2
-sarge==0.1.4
-semantic-version==2.4.2
-sockjs-tornado==1.0.1
-speaklater==1.3
-tornado==4.0.1
-unidecode==0.04.19
-watchdog==0.8.3
-webassets==0.11.1
-Werkzeug==0.8.3
+werkzeug==0.16.1
```

Comment:
To ease maintainance, would it be relevant to sort these packages?
Why not start file with a one line comment like "# Generated by pip freeze | grep -v excluded-package | sort" so that it becomes easier/obvious to update to next version?

Reply:
Just commented where this comes from and that it is ordered as the source in setup.py